airflow/ui/src/pages/DagsList/Dag/Header.tsx

```diff
@@ -118,3 +118,5 @@ export const Header = ({
     </Box>
   );
 };
+
+// test
```

Comment:
Just a comment to remember to remove it :)

Reply:
BTW. It will be difficult to check this way - because breeze changes generate "test all" condition. but you can test selective checks very easily providing a commit hash.

````
breeze ci selective-check --commit-ref 9c4b81d71f3caee3a3b1c6d9c626ee5d16f0db7c
````

Before your change:

![image](https://github.com/user-attachments/assets/eaa12d88-93fc-4954-931e-43de3c82d51c)


After:

![image](https://github.com/user-attachments/assets/1ec9d797-e9b1-4e11-a05e-341699c6d475)